README.md

```diff
@@ -7,29 +7,37 @@ See video: https://www.youtube.com/watch?v=MbYySNSVVFo
 
 Capture sound waves with microphone of computer and then represent this wave on leds of arduino using Pingo.io.
 
-## How i install/use this project?
+## What do you need?
+The hardware used is an Arduino Uno and the components are:
+- 6 120 Ω resistors;
+- 6 3mm LEDs;
+- Some jumpers;
+- Some noise.
 
+## How i install/use this project?
 When you use pingo/pyfirmata we need preload StandardFirmata on your board, on ARDUINO IDE preload this example on menus:
 
 ```"File" -> "Examples" -> "Firmata" -> "StandartFirmata"```
 
-We need connect cables and leds on your Arduino :
 
-![Arduino](/contrib/draw.png "Arduino")
+## Arduino Setup
```

Comment:
Arduino Setup title is not necessary or change to more descriptive title. Please.